pkg/kn/commands/source/container/create.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package container
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"knative.dev/client/pkg/kn/commands/flags"
+	knflags "knative.dev/client/pkg/kn/flags"
+
+	"knative.dev/client/pkg/kn/commands"
+	"knative.dev/client/pkg/sources/v1alpha2"
+	eventingv2 "knative.dev/eventing/pkg/apis/sources/v1alpha2"
+)
+
+// NewContainerCreateCommand for creating source
+func NewContainerCreateCommand(p *commands.KnParams) *cobra.Command {
+	var podFlags knflags.PodSpecFlags
+	var sinkFlags flags.SinkFlags
+
+	cmd := &cobra.Command{
+		Use:   "create NAME --image IMAGE",
+		Short: "Create a container source",
+		Example: `
+  # Create a ContainerSource 'containersrc' to instantiate an image 'docker.io/sample/image' that can generate events
+  kn source container create containersrc --image docker.io/sample/image`,
+
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			if len(args) != 1 {
+				return errors.New("requires the name of the source to create as single argument")
+			}
+			name := args[0]
+
+			// get client
```

Comment:
Comment not needed and disagrees with code which is `new...`

Reply:
removed. Thanks.